Libraries/ActionSheetIOS/React-RCTActionSheet.podspec

```diff
@@ -24,7 +24,7 @@ Pod::Spec.new do |s|
   s.documentation_url      = "https://reactnative.dev/docs/actionsheetios"
   s.license                = package["license"]
   s.author                 = "Facebook, Inc. and its affiliates"
-  s.platforms              = { :ios => "10.0", :tvos => "10.0", :osx => "10.13" } # TODO(macOS GH#214)
```

Comment:
ios=>"13.0" it will make our sizes small too...

Reply:
I agree that we should try to do that ASAP, but I'm not sure we necessarily want to diverge from upstream here. it would be weird if someone consumed this fork of react-native for x-plat support but that resulted in a different minimum supported OS than the upstream repo.

Separate of this change it might be good to try to push upstream to be more aggressive with minimum supported OS given usage trends and practical impact of size as well as work internally to see if we can override this in our consumption of react-native-macos (and in turn provide guidance to other clients who want to upgrade their minimum supported OS before we are able to get a newer version pushed to upstream)